package.json

```diff
@@ -13,8 +13,8 @@
     "_check:links": "HTMLTEST_ARGS='--log-level 1' npm run __check:links",
     "_check:links:internal": "npm run __check:links",
     "_check:links--warn": "npm run _check:links || (echo; echo 'WARNING: see link-checker output for issues.'; echo)",
-    "_check:spelling": "npx cspell -c .vscode/cspell.json content data",
-    "_check:text": "npx textlint content data",
+    "_check:spelling": "npx cspell --no-progress -c .vscode/cspell.json content",
+    "_check:textlint": "npx textlint content",
```

Comment:
Nit: what is being checked is the "text", so I have a slight preference for the old script name.

Reply:
fixed